charts/datastore/templates/deployment.yaml

```diff
@@ -49,28 +55,36 @@ spec:
 {{- end }}
 {{- end }}
 {{- end }}
+{{- if (eq .Values.datastore.inMemoryDb true) }}
+            - name: ROOKOUT_DOP_IN_MEMORY_DB
+              value: "True"
+{{- end }}
 {{- if (.Values.datastore.url_prefix) }}
             - name: ROOKOUT_DOP_URL_PREFIX
               value: {{ .Values.datastore.url_prefix | quote }}
 {{- end }}
             - name: ROOKOUT_DOP_SERVER_MODE
               value: {{ .Values.datastore.serverMode | quote }}
+            - name: ROOKOUT_DOP_PORT
+              value: .Values.datastore.dopContainerPort
```

Comment:
I think this is a mistake, the ROOKOUT_DOP_PORT determines the outside port, not the inner one